src/pal/src/exception/seh-unwind.cpp

```diff
@@ -135,8 +139,11 @@ BOOL VirtualUnwind(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *contextPoint
     unw_context_t unwContext;
     unw_cursor_t cursor;
 
-    // Set the unwind context to the specified windows context
-    WinContextToUnwindContext(context, &unwContext);
+    st = unw_getcontext(&unwContext);
```

Comment:
I'm working through this right now, it seems that unw_step isn't updating the register windows for me properly, so I'll need to do a bit of digging to see what works best on linux and mac.